collector/transport_node_collector.go

```diff
@@ -108,3 +127,23 @@ func (c *transportNodeCollector) buildTransportZoneEndpointLabels(transportZoneE
 	}
 	return labels
 }
+
+func (c *transportNodeCollector) initEdgeClusterMap() error {
```

Comment:
btw if edgeClusterMap only has specific use (not across struct), can we put it as return value instead?

Reply:
this is true let me do that